containers/streamsets/Dockerfile

```diff
@@ -15,6 +15,10 @@ RUN sudo chmod +x /tmp/configure_image.sh && /tmp/configure_image.sh
 COPY kafka-clients-2.2.2.jar ${SDC_DIST}/streamsets-libs/streamsets-datacollector-apache-kafka_2_0-lib/lib/kafka-clients-2.0.1.jar
 COPY streamsets-datacollector-kafka_multisource-protolib-3.18.0.jar  ${SDC_DIST}/streamsets-libs/streamsets-datacollector-apache-kafka_2_0-lib/lib/streamsets-datacollector-kafka_multisource-protolib-3.18.0.jar
 
+## This is needed for run-test-pipeline command
+CMD mkdir -p /home/sdc/tmp/
```

Comment:
First, CMD in Dockerfile is the main process that runs the container, for stuff like this RUN should be used.
Second, mkdir here shouldn't be necessary because according to the docs https://docs.docker.com/engine/reference/builder/#copy if you do COPY and if <dest> doesn’t exist, it is created along with all missing directories in its path.

Reply:
Fixed. Thanks